### DISCUSSION TITLE
Disable window wobbling

### DISCUSSION THREAD

**Discussion by andrinbr** [Eugeny/tabby#5457] [Q&A] [answered]
Is there a way to disable the window wobbling when you're using tab? (pressing Tab when there is no completion available from the shell)
I personally find it kind of annoying.

**Comment by Eugeny** [comment, marked as answer]
Yes, switch the _bell_ from visual to audible or off in Settings -> Terminal.

**Reply by andrinbr** [reply]
Thank you, I forgot about this one.

**Comment by lawweiliang** [comment]
@andrin55  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
